ui/components/app/beta-header/index.js

```diff
@@ -32,8 +32,9 @@ const BetaHeader = () => {
       className="beta-header"
       alignItems={AlignItems.center}
     >
-      <Typography
-        variant={TypographyVariant.H7}
+      <Text
+        variant={TextVariant.bodySm}
+        as="h6"
```

Comment:
We may be able to remove this if it doesn't break any e2e tests. Another contributor made a point that `h6` isn't semantically correct for all text tags
```suggestion
```